designs/index.md

```diff
@@ -143,7 +152,7 @@ for new documents. It will help you structure the document and create a visual
 consistency with other Bazel related documents. To do that, click on "Make a
```

Comment:
For consistency, use the [Bazel design doc template](
https://docs.google.com/document/d/1cE5zrjrR40RXNg64XtRFewSv6FrLV6slGkkqxBumS1w/edit). It includes the necessary header and creates a visual consistency with other Bazel related documents. To do that, click **File** > **Make a copy** or click this link to [make a copy of the design doc template](https://docs.google.com/document/d/1cE5zrjrR40RXNg64XtRFewSv6FrLV6slGkkqxBumS1w/copy).

Reply:
Done.